1-org/modules/centralized-logging/variables.tf

```diff
@@ -25,15 +25,27 @@ variable "resources" {
 }
 
 variable "resource_type" {
-  description = "Resource type of the resource that will export logs to destination. Must be: project, organization, or folder."
+  description = "Resource type of the resource that will export logs to destination. Must be: project, organization, billing_account or folder."
   type        = string
 
   validation {
-    condition     = contains(["project", "folder", "organization"], var.resource_type)
-    error_message = "The resource_type value must be: project, organization, or folder."
+    condition     = contains(["project", "folder", "organization", "billing_account"], var.resource_type)
```

Comment:
I think we should remove `billing_account` from here, since we are enabling/disabling the billing account sink though `enable_billing_account_sink` variable.
If the user calls our `centralized-logging` module passing `resource_type = "billing_account"` an error will be raised due to a non-match with `parent_resource_id` (it won't be the billing account id, but the folder or org id).